### PR TITLE
About page

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
It's the about page!

## Additions

- Added `about-us` page -- a landing page for the entire about section

## Changes

- Updated Activity Snippets to include newsroom

## Testing

- Visit `/about-us/`

## Review

- @jimmynotjim 
- @anselmbradford 
- @sebworks 

## Preview

image

[Preview this PR without the whitespace changes](?w=0)

## Notes

- Events should be added to the activity snippets when that is ready

### CODE REVIEW

File: src/about-us/index.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        <h1>About Us</h1>
+        <p class="h3">
+            We're a new government agency created
+            after the financial crisis of 2008.
+            We write and enforce rules to ensure that markets for
+            consumer financial products and services are fair and open,
+            and to make financial information easier to understand.
+            We also build tools and resources that help arm consumers
+            with the information they need,
+            and empower consumers to make the best financial decisions
```
Comment (jimmynotjim) [inline src/about-us/index.html]: Missing period after "decisions"

File: src/about-us/index.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                    </div>
+                    <div class="media_body">
+                        <span class="h6">Director</span>
+                        <h3 class="h2 bureau-bio_name">Richard Cordray</h3>
+                        <a class="jump-link jump-link__right" href="about-rich-cordray">
+                            Cordray's bio
+                        </a>
+                    </div>
+                </div>
+                <div class="media bureau-bio u-mb30">
```
Comment (jimmynotjim) [inline src/about-us/index.html]: Do we take Deputy Director Antonakes out now or wait?

File: src/about-us/index.html [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+            </section>
+        </div>
+        <div class="content-l_col-1-2">
+            <section class="block
+                        block__border-top
+                        block__padded-top
+                        block__flush-bottom">
+                <h2>Strategy & Budget</h2>
+                <p>
+                    We work for the American public.
```
Comment (jimmynotjim) [inline src/about-us/index.html]: This reads very oddly and doesn't encourage someone to take a next step. @sarahsimpson09 any ideas to improve engagement and clarity here?

File: src/about-us/index.html (hunk 2 above)
Comment (KimberlyMunoz) [inline src/about-us/index.html]: Let's wait. We should match what's on `/the-bureau/` for the landing page.

File: src/about-us/index.html (hunk 1 above)
Comment (KimberlyMunoz) [inline src/about-us/index.html]: Good catch. Will fix with the next batch of reviews.

File: src/about-us/index.html [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+                        block__flush-bottom">
+                <h2>CFPB Leadership</h2>
+                <div class="media bureau-bio u-mb30">
+                    <div class="media_image-container">
+                        <img class="media_image" src="/static/img/cordray_round_360.jpg">
+                    </div>
+                    <div class="media_body">
+                        <span class="h6">Director</span>
+                        <h3 class="h2 bureau-bio_name">Richard Cordray</h3>
+                        <a class="jump-link jump-link__right" href="about-rich-cordray">
```
Comment (jimmynotjim) [inline src/about-us/index.html]: Link should be `/the-bureau/about-rich-cordray/`

File: src/about-us/index.html [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+                    </div>
+                </div>
+                <div class="media bureau-bio u-mb30">
+                    <div class="media_image-container">
+                        <img class="media_image" src="/static/img/antonakes_round_360.jpg">
+                    </div>
+                    <div class="media_body">
+                        <span class="h6">Deputy Director</span>
+                        <h3 class="h2 bureau-bio_name">Steve Antonakes</h3>
+                        <a class="jump-link jump-link__right" href="about-steve-antonakes">
```
Comment (jimmynotjim) [inline src/about-us/index.html]: Link should be `/the-bureau/about-steve-antonakes/`

**Comment by jimmynotjim** [comment]
Make sure to update the header nav with a link to this page

File: src/about-us/index.html (hunk 3 above)
Comment (KimberlyMunoz) [inline src/about-us/index.html]: According to GHE/flapjack/about/issues/2#issuecomment-49481, this is basically placeholder.

Comment (schaferjh) [inline src/about-us/index.html]: I'm not sure exactly which line @jimmynotjim thinks reads oddly? 

This content is in pre-clearence so it's not totally placeholder, but there is some wiggle room if something seems horribly horrible.

Comment (KimberlyMunoz) [inline src/about-us/index.html]: I think it's 

```
We work for the American public.
We publish our strategy and budget
so you can understand our priorities.
```

Any chance we could hop into a Google chat hangout to hash out what needs to be done for this PR?

Comment (jimmynotjim) [inline src/about-us/index.html]: Yeah, it sounds...forced and fake...like the Zalinksy ad in Tommy Boy https://www.youtube.com/watch?v=jRHULPFE3zo

**Comment by KimberlyMunoz** [comment]
Pushed updates for a majority of @jimmynotjim's review feedback. @sarahsimpson09 and @schaferjh, do y'all know how what can be next steps [for this](https://github.com/cfpb/cfgov-refresh/pull/782#discussion_r35026407)?

**Comment by sebworks** [comment]
:+1:

**Comment by jimmynotjim** [comment]
:+1: I'd say go with the content, we can always change it later.

**Comment by sarahsimpson09** [comment]
I am hesitant to change that language because it is already in
preclearance. Plus, James worked on developing it so I'm not really sure
what the process was as far as who wanted that language there since he did
most of that work. I do know that the language that's there took a while to
nail down and create buy-in for, so we may not want to change it too much
so we don't have to do it again (depending on what clearance says). As for
more of a "call to action" with those links, we can add verbs to help. Try:

Read our strategic plan
Browse financial reports and updates
Learn more about our strategy and budget

**Pull request merged**